packages/editable-commands/src/listeners/pluginMessageUpdate.ts

```diff
@@ -0,0 +1,25 @@
+import { Events, Listener, PieceContext } from '@sapphire/framework';
+import type { Message } from 'discord.js';
+
+export class PluginListener extends Listener<typeof Events.MessageUpdate> {
+	public constructor(context: PieceContext) {
+		super(context, { event: Events.MessageUpdate });
+	}
+
+	public run(old: Message, message: Message) {
+		// If the contents of both messages are the same, return:
+		if (old.content === message.content) return;
+
+		// If the message was sent by a webhook, return:
+		if (message.webhookId !== null) return;
+
+		// If the message was sent by the system, return:
```

Comment:
Which system? Goliath? Amazon??

Reply:
Zuckerbook, obviously

Reply:
That requires us to develop a machine learning program that identifies lamps.